### PR TITLE
RS: Update log rotation

### PR DESCRIPTION
**Pull request by rrelledge**
[Staged preview](https://docs.redis.com/staging/DOC-2593/rs/clusters/logging/log-security/)

### CODE REVIEW

File: content/rs/clusters/logging/log-security.md [hunk 1]
```diff
@@ -16,48 +16,68 @@ aliases: [
...
+## Send logs to a remote logging server
 
 Redis Enterprise sends logs to syslog by default. You can send these logs to a remote logging server by configuring syslog.
 
 To do this, modify the syslog or rsyslog configuration on your operating system to send logs in `/var/opt/redislabs/log` to a remote monitoring server of your choice.
 
 ## Log rotation
 
-Redis Enterprise uses the default logrotate daemon to schedule rotation of logs stored on the operating system. The configuration of log rotation may be found at /etc/logrotate.d.
+Redis Enterprise uses the default `logrotate` daemon to schedule rotation of logs stored on the operating system. You can find the log rotation configuration file at `$pkgconfdir/logrotate.conf` as of Redis Enterprise Software version 7.2 (`pkgconfdir` is `/opt/redislabs/config` by default, but can be changed in a custom installation).
```
Comment (zviRosenfeldRedis) [inline content/rs/clusters/logging/log-security.md:27]: logrotate isn't ran by the default logtorate daemon anymore. In 7.2 out own job scheduler runs it every 5 minutes.

File: content/rs/clusters/logging/log-security.md [hunk 2]
```diff
@@ -16,48 +16,68 @@ aliases: [
...
 
-By default the log rotation should occur on a daily basis. We recommend that you send log files to a remote logging server so that they can be more effectively maintained.
+By default, log rotation occurs every five minutes or when logs exceed 200 MB. We recommend sending log files to a remote logging server so you can maintain them more effectively.
```
Comment (zviRosenfeldRedis) [inline content/rs/clusters/logging/log-security.md:29]: There's a confusion here. logrotate tool is ran every 5 minutes (by the job scheduler). The logrotate tool examines every log file and rotates each log that exceeds 200MB.

File: content/rs/clusters/logging/log-security.md [hunk 3]
```diff
@@ -16,48 +16,68 @@ aliases: [
...
 ```
 
-Below describes what the log rotation this configuration policy puts into effect.
+- `/var/opt/redislabs/log/*.log` - `logrotate` checks the files under directory `/var/opt/redislabs/log/` and rotates any files that end with the extension `.log`.
 
-- `/var/opt/redislabs/log/*.log` - When logrotate runs it checks the files under directory `/var/opt/redislabs/log/` and rotates any files that end with the extension .log.
+- `size 200M` - Rotate log files that exceed 200 MB.
 
-- Daily - The interval is set to daily.
+- `daily` - The interval is set to daily.
```
Comment (zviRosenfeldRedis) [inline content/rs/clusters/logging/log-security.md:69]: We no longer have the `daily` attribute in this section of the logrotate's configuration

File: content/rs/clusters/logging/log-security.md [hunk 4]
```diff
@@ -16,48 +16,68 @@ aliases: [
...
 
-- Missingok - If there are missing logfiles don't do anything.
+- `missingok` - If there are missing log files, do nothing.
 
-- Copytruncate - Truncate the original log file to zero sizes after creating a copy.
+- `copytruncate` - Truncate the original log file to zero sizes after creating a copy.
 
-- rotate 7 - Keep 7 log files and delete the rest.
+- `rotate 30` - Keep 30 log files and delete the rest.
```
Comment (zviRosenfeldRedis) [inline content/rs/clusters/logging/log-security.md:75]: This section now reads `rotate 2000`. Meaning we keep up to 2000 files. That's just logrotat's way to say infinite. We actually keep logs for 7 days (defined by `maxage 7`)

File: content/rs/clusters/logging/log-security.md [hunk 5]
```diff
@@ -16,48 +16,68 @@ aliases: [
...
 
-- compress - gzip log files.
+- `compress` - gzip log files.
```
Comment (zviRosenfeldRedis) [inline content/rs/clusters/logging/log-security.md:77]: I think there should be a section for `maxage 7` too:
- `maxage 7` - Keep the rotated log files for 7 days.

File: content/rs/clusters/logging/log-security.md (hunk 1 above)
Comment (rrelledge) [reply content/rs/clusters/logging/log-security.md]: Fixed

File: content/rs/clusters/logging/log-security.md (hunk 2 above)
Comment (rrelledge) [reply content/rs/clusters/logging/log-security.md]: Thanks for clarifying. That makes more sense.

File: content/rs/clusters/logging/log-security.md (hunk 3 above)
Comment (rrelledge) [reply content/rs/clusters/logging/log-security.md]: Removed

File: content/rs/clusters/logging/log-security.md (hunk 4 above)
Comment (rrelledge) [reply content/rs/clusters/logging/log-security.md]: Updated `rotate 2000` and added `maxage 7`

File: content/rs/clusters/logging/log-security.md [hunk 6]
```diff
@@ -16,48 +16,68 @@ aliases: [
...
+    copytruncate
+    rotate 30
+    compress
+    notifempty
+    nodateext
 }
 ```
 
-Below describes what the log rotation this configuration policy puts into effect.
+- `/var/opt/redislabs/log/*.log` - `logrotate` checks the files under directory `/var/opt/redislabs/log/` and rotates any files that end with the extension `.log`.
```
[the remark this answers is not in the document]
Comment (rrelledge) [reply content/rs/clusters/logging/log-security.md]: Added more info for these two config sections

**Review by zviRosenfeldRedis — APPROVED** [review]

**Pull request merged**